PSPDFKitPlugin/PSPDFKitPlugin.m

```diff
@@ -77,10 +77,16 @@ - (void)setOptions:(NSDictionary *)options forObject:(id)object animated:(BOOL)a
                     [self setValue:newOptions[key] forKey:[key stringByAppendingFormat:@"For%@WithJSON", [object class]]];
                 }
                 else {
-                    //use KVC
-                    setter = [prefix stringByAppendingString:@":"];
-                    if ([object respondsToSelector:NSSelectorFromString(setter)]) {
-                        [object setValue:newOptions[key] forKey:key];
+                    // Try the super class. For example, we try PSPDFDocument methods for Image Documents.
+                    setter = [prefix stringByAppendingFormat:@"For%@WithJSON:", [object superclass]];
```

Comment:
This is some code duplication that we could solve with an inline block + parameter.

Reply:
Good point! Addressed!